### ISSUE TITLE
Mutiple search domain needs to be split

### ISSUE THREAD

**Issue by zibok** [adrienverge/openfortivpn#504]
I'm using the 1.11.0 release against my corporate VPN, which pushes 2 domains in the search list.

This push works normally with the official client on iOS for instance, and for our Windows clients, but with openfortivpn, it adds on `search domain1.tld;domain2.tld`

I assume this means openfortivpn receives this string and it needs to be splitted.

```
INFO:   Adding VPN nameservers...
DEBUG:  Adding "nameserver <ip1>", to /etc/resolv.conf.
DEBUG:  Adding "nameserver <ip2>", to /etc/resolv.conf.
DEBUG:  Adding "search domain1.tld;domain2.tld", to /etc/resolv.conf.
```

**Comment by mrbaseman** [comment]
Actually, the dns-suffix setting is simply a string in the configuration on the Fortigate, and it would be no problem to `set dns-suffix "domain1.tld domain2.tld"` - but since the official client seems to replace semicolon by spaces we should do the same. I have created a pull request that implements this.

**Comment by zibok** [comment]
Thanks. I am not enough comfortable with C to propose one.

**Comment by zibok** [comment]
Just tried with a version compiled from master, and it solves my issue. Thanks for your reactivity, it's awesome!

**Comment by mrbaseman** [comment]
the fix went into [release 1.12.0](https://github.com/adrienverge/openfortivpn/releases/tag/v1.12.0)
